SECURITY.md

```diff
@@ -0,0 +1,68 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+The OPI security policy aims to efficiently handle security vulnerabilities
+while minimizing public disclosure until a fix is available.
+
+Reporters are requested to submit security issues privately via the
+[GitHub advisory](https://github.com/opiproject/opi/security/advisories/new)
+system.
+
+Reports should be acknowledged within 5 days. OPI maintainers will try to
+respond to a report within a maximum of 10 days, indicating the next steps
+in handling the submission (although it will depend on the severity of the
+report).
+
+## Reporting a Vulnerability in 3rd Party Modules
+
+Security issues in third-party modules should be reported according to their
+own security policies or directly to their respective maintainers if no such
+security policy is in place for the respective module.
+
+## Vulnerability Disclosure Policy
+
+Upon receiving a security report, OPI follows the Common Vulnerabilities and
+Exposures (CVE®) process outlined below:
+
+1. OPI maintainer acknowledges the receipt of the report and calls for a
+maintainer/security experts meeting to discuss the scope of the vulnerability
+and identify any potentially related problems.
```

Comment:
Should the maintainer(s)/security experts identify the risk of the issue and assign a severity level at this point to determine the priority that the issue needs to be worked?

Reply:
Definitely, but I think this level of details (internal handling) does not need to be described here. I would rather keep it as short as possible. The severity/priority will influence other points like finding resources etc.

Reply:
That is reasonable and we can keep that as an internal aspect.  Updates all look good.